src/mudlet.cpp

```diff
@@ -4741,7 +4823,16 @@ void mudlet::setShowIconsOnMenu(const Qt::CheckState state)
 
 void mudlet::setInterfaceLanguage(const QString& languageCode)
 {
-    mInterfaceLanguage = languageCode;
+    if (mInterfaceLanguage != languageCode) {
+        mInterfaceLanguage = languageCode;
+        loadTranslators(languageCode);
+        // For full dynamic language change support (no restart necessary) we
+        // would also need a call here to do the same in this class that the
+        // signal_guiLanguageChanged call will do in classes wired up to it
+        // {run retranslateUi(), and regenerate (persistent) texts generated
+        // within the Mudlet application code}...
```

Comment:
I've seen what that does to our code and it is _nasty_... I'd rather not have it.

At the end of the day it is a minor, once-off convenience for the user while a really big turnoff for the developer at every step of the way.

Reply:
Fair enough - though it *is* nice for debugging translations as you wouldn't need to restart to test each different translation.